examples/blog/index.html

```diff
@@ -5,6 +5,8 @@
     <title>React-Admin Blog Sample Application</title>
 </head>
 <body>
+    <div id="my-app"></div>
```

Comment:
The app was renderer un the `body` but since notifications are also in the body, react delete them automatically.